recipe/0036-composition-from-multiple-images/manifest.json

```diff
@@ -0,0 +1,64 @@
+{
+    "@context": "http://iiif.io/api/presentation/3/context.json",
+    "id": "{{ id.url }}",
+    "type": "Manifest",
+    "label": { "en": [ "Folio from Grandes Chroniques de France, ca. 1460" ] },
+    "items": [
+        {
+            "id": "https://bvmm.irht.cnrs.fr/iiif/4490/canvas/p1",
```

Comment:
```suggestion
            "id": "{{ id.path }}/canvas/p1",
```

Reply:
Great, that's working: https://codesandbox.io/s/0036-composition-from-multiple-images-5h5712?file=/index.html 🎉  thanks!